coral/src/app/layout/SidebarNavigation.tsx

```diff
@@ -0,0 +1,113 @@
+import { Box, Flexbox, Icon } from "@aivenio/design-system";
+import database from "@aivenio/design-system/dist/src/icons/database";
+import codeBlock from "@aivenio/design-system/dist/src/icons/codeBlock";
+import layoutGroupBy from "@aivenio/design-system/dist/src/icons/layoutGroupBy";
+import code from "@aivenio/design-system/dist/src/icons/code";
+import people from "@aivenio/design-system/dist/src/icons/people";
+import list from "@aivenio/design-system/dist/src/icons/list";
+import cog from "@aivenio/design-system/dist/src/icons/cog";
+import data from "@aivenio/design-system/dist/src/icons/console";
+import classes from "src/app/layout/SidebarNavigation.module.css";
+
+const originLocationKlawAngular = window.location.origin;
+function createSidebarListItem({
+  icon,
+  href,
+  linkText,
+  active = false,
+}: {
+  icon: typeof data;
+  href: string;
+  linkText: string;
+  active?: boolean;
+}) {
+  return (
+    <Flexbox
+      htmlTag={"li"}
+      alignItems={"center"}
+      marginBottom={"l1"}
+      colGap={"l1"}
+      className={
+        active
+          ? classes.primaryNavigationLinkActive
+          : classes.primaryNavigationLink
+      }
+      paddingLeft={"l2"}
+    >
+      <Icon icon={icon} />
+      <a href={href} aria-current={active && "page"}>
+        {linkText}
+      </a>
+    </Flexbox>
+  );
+}
+
+function SidebarNavigation() {
+  return (
+    <Box
+      component={"nav"}
+      backgroundColor={"grey-0"}
+      aria-label={"Primary navigation"}
```

Comment:
I suggest changing this to `Main navigation`, as primary has quite engineery vibes to me. Also, what do you think if the name of the component should be the same as `aria-label`? _SidebarNavigation_ vs _Primary navigation_

Reply:
I can change the name! Maybe we can also make a housekeeping issues to double check all copies (including content for assistive technology) with a technical writer! Perkins School of Blinds site (it's one I like to check because they do great work with their site) for example has really descriptive names for their navigations, e.g "Top Global Links" or "Visit Other Perkins Sites". I really like that! It's a low effort improvement. 

I can rename the component, too. Even if we would change the name of the label, `MainNavigation` still is a better information for the component name.